### DISCUSSION TITLE
[Feedback] Add L2 predeploy contract addresses to the document

### DISCUSSION THREAD

**Discussion by Chomtana** [ethereum-optimism/developers#61] [Feedback]
### Feedback

I think we should have this L2 predeploy contract address table in the document

https://github.com/ethereum-optimism/optimism/blob/develop/specs/predeploys.md

<img width="471" alt="image" src="https://github.com/ethereum-optimism/developers/assets/4103490/201cee9f-e0d2-4cdb-8ea7-5bdb5d247f03">

**Comment by sbvegan** [comment]
cc @cpengilly

**Comment by opfocus** [comment]
great progress

**Comment by cpengilly** [comment]
thank you @sbvegan which page/document are we wanting to add this table?

**Comment by cpengilly** [comment]
@Chomtana thanks for the feedback! which page in the docs would you like me to add this table to? I apologize but seemed to have missed that context somewhere :)

**Comment by opfocus** [comment]
Get L2 contract addresses
Addresses of system contracts on the L2 side of the network are the same on every network. We provide these addresses as constants within the Lib_PredeployAddresses contract.    https://docs.optimism.io/dapp-developers/contracts/system-contracts     Highly relevant here

**Reply by Chomtana** [reply]
Look good to have it on that page

**Reply by cpengilly** [reply]
got it! thanks!

**Reply by Chomtana** [reply]
<img width="445" alt="image" src="https://github.com/ethereum-optimism/developers/assets/4103490/6b732933-ab8c-42f1-96b2-0730b5f22be2">

URL is changed to https://docs.optimism.io/chain/addresses

**Comment by Chomtana** [comment]
It's now available at https://docs.optimism.io/chain/addresses
